iSeg/bmp_read_1.cpp

```diff
@@ -136,16 +136,15 @@ bmphandler::bmphandler()
 	sliceprovide_installer = SliceProviderInstaller::getinst();
 	stackcounter = 1;
 	mode1 = mode2 = 1;
-	return;
+	return; // ?
```

Comment:
...

Reply:
@sanderegg yes, I also wondered why that was placed there. ugly stuff